lib/ansible/modules/cloud/amazon/sns.py

```diff
@@ -191,9 +191,11 @@ def main():
 
     sns_kwargs = dict(
         Message=module.params['msg'],
-        Subject=module.params['subject'],
-        MessageStructure=module.params['message_structure'],
+        MessageStructure=module.params['message_structure']
```

Comment:
```suggestion
        MessageStructure=module.params['message_structure'],
```

The final trailing comma is optional, yes, but it makes diffs cleaner.